webui/react/src/pages/TrialDetails/TrialDetailsMetrics.tsx

```diff
@@ -72,7 +60,13 @@ const TrialDetailsOverview: React.FC<Props> = ({ experiment, trial }: Props) =>
 
   const trials: (TrialDetails | undefined)[] = useMemo(() => [trial], [trial]);
 
-  const { metrics, data: allData, scale, setScale } = useTrialMetrics(trials);
+  const {
+    metrics,
+    isLoaded: metricsLoaded,
```

Comment:
**super nit**: `isMetricsLoaded` for me personally makes it more obvious that it is a boolean state but no hard feelings if you don't want to.

Reply:
✅